lib/config/migrations/custom/semantic-prefix-migration.spec.ts

```diff
@@ -0,0 +1,30 @@
+import { SemanticPrefixMigration } from './semantic-prefix-migration';
+
+describe('config/migrations/custom/semantic-prefix-migration', () => {
+  it('should work', () => {
+    expect(SemanticPrefixMigration).toMigrate(
+      {
+        semanticPrefix: 'fix(deps): ',
+      } as any,
+      { semanticCommitType: 'fix', semanticCommitScope: 'deps' }
+    );
+  });
+
+  it('should remove non-string values', () => {
+    expect(SemanticPrefixMigration).toMigrate(
+      {
+        semanticPrefix: true,
+      } as any,
+      {}
+    );
+  });
+
+  it('should migrate prefix with no-scope to null', () => {
+    expect(SemanticPrefixMigration).toMigrate(
+      {
+        semanticPrefix: 'fix: ',
+      } as any,
+      { semanticCommitType: 'fix', semanticCommitScope: null }
+    );
+  });
+});
```

Comment:
Could you please also add test case for random string without colon, for example:
```ts
{
  semanticPrefix: 'test'
}
```